ministryofjustice/correspondence_tool_staff
# CDPT-71 Add request method options to SAR creation and display

## Description
When a London Disclosure user creates a SAR, they should be able to specify how the SAR was received by MOJ

## Self-review checklist



### Screenshots
![image](https://user-images.githubusercontent.com/1161161/176559060-6231af99-43d0-4e76-85d7-feb80b0fdc4a.png)

![image](https://user-images.githubusercontent.com/1161161/176559145-52396921-ba60-436d-af46-0a87721f277e.png)

![image](https://user-images.githubusercontent.com/1161161/176559319-b073c1a5-4cd3-4326-9086-c474cc36c409.png)

![image](https://user-images.githubusercontent.com/1161161/176559348-1c6d46e2-123b-4f09-a708-f424780b8708.png)

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CDPT-71
https://dsdmoj.atlassian.net/browse/CDPT-89

### Deployment
This PR contains a data migration - see the Readme. When it is deployed onto a namespace, run `rake data:migrate` to populate the SAR case request_method values.

### Manual testing instructions
Sign into the app as David Attenborough and create a new SAR case. You should be able to specify the request method from the options Post, Email, Web Portal and so on. 

When the case details page is displayed the correct request method should be displayed. 

When an Admin user creates a SAR case, the default settings should create a valid case using "Post" as the request method.
